core/iwasm/common/wasm_native.c

```diff
@@ -78,7 +78,7 @@ check_symbol_signature(const WASMType *type, const char *signature)
              && type->types[i] <= VALUE_TYPE_I32
              && sig == sig_map[type->types[i] - VALUE_TYPE_F64])
 #if WASM_ENABLE_REF_TYPES != 0
-            || (sig == 'i' && type->types[i] == VALUE_TYPE_EXTERNREF)
+            || (sig == 'I' && type->types[i] == VALUE_TYPE_EXTERNREF)
```

Comment:
Seems that both 'i' and 'I' are not good, how about changing to 'r' to represent a ref (also can represent other refs for GC feature), and update the document export_native_api.md?

Reply:
If we are going to treat it (refs) as a pointer, `*` should be OK, and we can run an address check.
Using `r` also means we won't do an address check like a pointer of an object.  Is it OK ?